web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java

```diff
@@ -788,6 +787,12 @@ private void assertCsvRecord(List<CSVRecord> csvList, ResourceType objType, PID
             assertEquals(permissions, rec.get(ExportCsvService.PATRON_PERMISSIONS_HEADER));
             assertEquals(embargoDate, rec.get(ExportCsvService.EMBARGO_HEADER));
             assertEquals(behavior, rec.get(ExportCsvService.VIEW_BEHAVIOR_HEADER));
+
+            if (accessSurrogate == null) {
```

Comment:
Was this left over from another PR? It's probably fine here, I was just kind of surprised to see it as new

Reply:
thanks for catching!